src/context/EarnManagerProvider/providers/yearn/components/YearnManager/YearnDeposit.tsx

```diff
@@ -76,7 +78,9 @@ const initialState: YearnDepositState = {
   deposit: {
     fiatAmount: '',
     cryptoAmount: '',
-    slippage: ''
+    slippage: '',
+    txStatus: 'pending',
+    usedGasFee: ''
```

Comment:
these should live on the `YearnDepositValues` vs field values since those are specific to the deposit form

Reply:
Done.